starport/interface/cli/starport/cmd/network_chain_join.go

```diff
@@ -28,13 +39,21 @@ func NewNetworkChainJoin() *cobra.Command {
 		Args:  cobra.ExactArgs(1),
 	}
 	c.Flags().AddFlagSet(flagSetHomes())
+	c.Flags().String(flagGentx, "", fmt.Sprintf("Path to a gentx file (optional, requires --%s to be provided)", flagGentx))
```

Comment:
> fmt.Sprintf("Path to a gentx file (optional, requires --%s to be provided)", flagGentx))

`--peer` must be provided if `--gentx` is provided right? Therefore, we must `flagGentx` -> `flagPeer`

Reply:
solved by 90cc70a.